test/run-envoy-unit-tests.sh

```diff
@@ -1,5 +0,0 @@
-#!/bin/bash
-
-set -e
-
-vagrant ssh runtime -c "cd /home/vagrant/go/src/github.com/cilium/cilium ;make build; make tests-envoy"
```

Comment:
How will we test envoy from now on? Is there a plan to create a jenkins job for the new repo? cc @eloycoto

Reply:
No concrete plan yet, but obviously needs to be done.

Reply:
So far there are no code changes in the cilium/proxy repo, so we are good.